### PR TITLE
Quick Test Helper for Creating Blocks

### PR DESCRIPTION
**Pull request by rawfalafel**
Wrote a simple method to facilitate creating blocks in `core_test.go`

### CODE REVIEW

File: beacon-chain/blockchain/core_test.go [hunk 1]
```diff
@@ -727,30 +759,26 @@ func unique(ints []int) []int {
...
 
-func NewBlockWithSlotNumber(t *testing.T, slotNumber uint64) *types.Block {
-	b, err := types.NewBlock(&pb.BeaconBlockResponse{
-		SlotNumber: slotNumber,
-		ParentHash: make([]byte, 32),
-	})
-
-	if err != nil {
-		t.Fatalf("failed to instantiate block with slot number: %d", slotNumber)
+// NewBlock is a helper method to create blocks with sane defaults
```
Comment (terenc3t) [inline beacon-chain/blockchain/core_test.go]: s/sane/same

Comment (terenc3t) [reply beacon-chain/blockchain/core_test.go]: punctuation too

**Comment by rawfalafel** [comment]
updated!

**Comment by terenc3t** [comment]
Great. Thanks! 👍

**Pull request merged**